docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  app:
+    image: friendsofsymfony/fos-http-cache
```

Comment:
I agree with @dbu. If the Dockerfile becomes broken, contributors to this library should rather report it so that it is fixed instead of using an older image. Thus, the Travis build on their PR would be broken because of that anyway.